use_position_queue: True
duration: 2.0
# Points should be in gait-coordinates:
# x: forward
# y: upward
# z: sideways
points:
  - x: 0.1
    y: 0.0
    z: 0.45
  - x: 0.2
    y: 0.0
    z: 0.45
